### ISSUE TITLE
Set up search console verification

### ISSUE THREAD

**Issue by stichbury** [kedro-org/kedro#2616]
I'm trying to add a verification file to the docs so that I can use search console on the docs to see what query strings are bringing users to our pages.

Have been following this https://stackoverflow.com/questions/27748396/using-google-analytics-in-readthedocs-org 

* I generated a verification file
* Added it to `docs/source` and updated `conf.py`
* Rebuilt docs, but the file isn't in the root of the `html` folder

I think this is because `source` is in the exclusion list in `conf.py` and wondered if you have any thoughts @astrojuanlu on whether I can put the verification elsewhere and then move it into the root, or get around the exclusion list somehow?

[Here's my branch](https://github.com/kedro-org/kedro/tree/docs-add-search-console-verification)

**Comment by astrojuanlu** [comment]
Yeah, having `source` in `exclude_patterns` always looked wrong to me. I'll have a look at this during the week.

**Comment by astrojuanlu** [comment]
File is live at https://docs.kedro.org/en/stable/googlebce3ad2fda582ae8.html as per gh-2631, hopefully this can be closed?

**Comment by stichbury** [comment]
@astrojuanlu it's not working :( 

So the file needs to be at docs.kedro.org/google..... and because of the way the RTD folder structure is set up, obviously isn't (it's in the en/stable subdirectory). I set up a redirect in RTD but that doesn't seem to be working (https://docs.kedro.org/googlebce3ad2fda582ae8.html is returning page not found).

Please can you advise? Is there a way I can fake the location?

**Comment by astrojuanlu** [comment]
Merghhhh too bad. I see there are other verification methods available:

https://support.google.com/webmasters/answer/9008080?hl=en#choose_method

Maybe we can try HTML Meta tags or using the existing GA tracking code?

**Comment by astrojuanlu** [comment]
xref gh-2674

Do you still intend to explore the other methods? (Otherwise let's indeed keep the issue closed)
